azure-pipelines/vs-insertion-experimental.yml

```diff
@@ -17,6 +17,23 @@ resources:
     name: 1ESPipelineTemplates/1ESPipelineTemplates
     ref: refs/tags/release
 
+parameters:
+  - name: TargetBranch
+    type: string
+    default: main
+    displayName: 'Insertion Target Branch (select for manual insertion)'
+    values:
```

Comment:
I'm assuming these are all the versions that we currently support? And we will have to update this with releases too so it is up-to-date.

Reply:
Could you update [our release checklist](https://github.com/dotnet/msbuild/blob/main/documentation/release-checklist.md) with a new step to update the supported branches?

Reply:
The versions here are those that we expect to want to insert into sometimes which ~corresponds to the supported versions. Updating is mentioned w.r.t. to the insertion pipeline update in release checklist (I could add an explicit reference to the experimental pipeline too).

Reply:
I think an explicit mention to the experimental pipeline would be best. That list is extremely long and anything not explicitly called out will be missed during release.